src/models/gameRecord.js

```diff
@@ -35,6 +35,7 @@ const gameRecordSchema = new mongoose.Schema({
   missionHistory: [String],
   numFailsHistory: [Number],
   voteHistory: Object,
+  disableVoteHistory: Boolean,
```

Comment:
You'll also need to add to this line here in `finishGame()` to include the data:

https://github.com/vck3000/ProAvalon/blob/master/src/gameplay/game.ts#L1278-L1316

Reply:
Fixed this.

Reply:
I don't see this added yet?

Reply:
https://github.com/vck3000/ProAvalon/pull/509/files#diff-fa564d9f0a3077d128a9edae956714e418036440f018e4c42849d44fca697ab4R1315